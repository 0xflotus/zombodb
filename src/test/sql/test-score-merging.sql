-- this once caused a segfault during development
SELECT zdb.score(ctid) > 1.0 FROM events WHERE events ==> 'beer' OR events ==> 'wine';