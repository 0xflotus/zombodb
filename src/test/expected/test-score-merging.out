-- this once caused a segfault during development
SELECT zdb.score(ctid) > 1.0 FROM events WHERE events ==> 'beer' OR events ==> 'wine';
 ?column? 
----------
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
 t
(33 rows)

